ttyclock.c.orig

```diff
@@ -0,0 +1,682 @@
+/*
```

Comment:
this .orig file shouldn't show up here

Reply:
Ah must have been left over from a patch that didn't apply.